app/src/organisms/ErrorRecoveryFlows/shared/TwoColTextAndImage.tsx

```diff
@@ -14,25 +14,63 @@ import { RECOVERY_MAP } from '../constants'
 import { RecoverySingleColumnContentWrapper } from './RecoveryContentWrapper'
 import { RecoveryFooterButtons } from './RecoveryFooterButtons'
 
-import type { RecoveryContentProps } from '../types'
+import type { RecoveryContentProps, RouteStep } from '../types'
 
 export function TwoColTextAndImage(
   props: RecoveryContentProps
 ): JSX.Element | null {
-  const { routeUpdateActions, recoveryMap } = props
-  const { LOAD_LABWARE_SHUTTLE_AND_RETRY } = RECOVERY_MAP
-  const { route } = recoveryMap
-  const { proceedNextStep, goBackPrevStep } = routeUpdateActions
+  const { routeUpdateActions, recoveryMap, recoveryCommands } = props
+  const {
+    LOAD_LABWARE_SHUTTLE_AND_RETRY,
+    REPLACE_LABWARE_IN_HOPPER_AND_RETRY,
+    ROBOT_IN_MOTION,
+    MANUAL_LOAD_ON_SHUTTLE_AND_SKIP,
+  } = RECOVERY_MAP
+  const { route, step } = recoveryMap
+  const {
+    proceedNextStep,
+    goBackPrevStep,
+    handleMotionRouting,
+  } = routeUpdateActions
+  const { closeLabwareLatch } = recoveryCommands
   const { t } = useTranslation('error_recovery')
 
+  const reengageLatchRoutes: RouteStep[] = [
```

Comment:
```suggestion
  const REENGAGE_LATCH_ROUTES: RouteStep[] = [
```

Nit, let's also hoist constants outside of the lexical scope of the component.